samples/language/ConditionalBranching.qs

```diff
@@ -0,0 +1,42 @@
+/// # Sample
+/// Conditional Branching
+///
+/// # Description
+/// Q# supports three branching keywords: `if`, `elif`, and `else`. These behave in the normal
+/// way, if you're familiar with if expressions or statements in other languages. One key distinction
+/// in Q# is that ifs are expressions, not statements. That means that they can return values, and
+/// they effectively perform the job of both ternary expressions and if statements in other languages.
+/// If expressions allow your code to branch, or conditionally execute parts of a Q# program.
+namespace MyQuantumApp {
+    @EntryPoint()
+    operation Main() : Unit {
+        let number = 5;
+        // Conditionally messages "Fizz" if the `number`, in this case 5, is divisible by 3.
+        // Since 5 is not divisible by 3, the message "Fizz" will not be printed.
+        if number % 3 == 0 { Message("Fizz"); }
+
+        // Conditionally messages "Buzz" if the `number`, in this case 5, is divisible by 5.
+        // Since 5 is divisible by 5, the message "Buzz" will be printed.
+        if number % 5 == 0 { Message("Buzz"); }
+
```

Comment:
nit: I think the max number of empty lines between parts of the code should be 1.